### PR TITLE
#60 running queries

### PR DESCRIPTION
**Pull request by AVM82**
connect to #60

### CODE REVIEW

**Review by Shtramak — APPROVED** [review]
Nice regexps :)

File: src/main/resources/sql/populate.sql [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+VALUES
+    ('Alexandr', 'Nevskiy', 35)
+    ,('Name', 'Surname', 16)
+    ,('Irina', 'Ovechkina', 56)
+    ,('Petr', 'Petrov', 69)
+    ,('Ivan', 'Ivanov', 21)
+    ,('Olha', 'Polkovnikova', 21)
+    ,('IvanV', 'Ivanov', 22)
+    ,('IvanVV', 'Ivanov', 23)
+    ,('Antonina', 'Tishenko', 24);
```
Comment (kortov) [inline src/main/resources/sql/populate.sql:12]: I think in case of the first task it's better to fill values with Boundary Value Analysis (BVA): 15,16,17,34,35,36 https://goo.gl/wQmkYn

File: src/main/resources/sql/select2-1-select2-5.sql [hunk 2]
```diff
@@ -0,0 +1,35 @@
+-- Find the names of buyers, whose age is from 16 to 35
+SELECT c.name
```
Comment (kortov) [inline src/main/resources/sql/select2-1-select2-5.sql:2]: May be we should also select c.age in order to check results?

File: src/main/resources/sql/select2-1-select2-5.sql [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+FROM customers c
+WHERE c.age BETWEEN 16 AND 35;
+
+-- Find the names of buyers whose surname ends in 'OV'
+SELECT c.name
```
Comment (kortov) [inline src/main/resources/sql/select2-1-select2-5.sql:7]: May be we should also select c.surname in order to check results?

**Review by kortov — APPROVED** [review]

File: src/main/resources/sql/select2-1-select2-5.sql [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+FROM customers c
+WHERE c.surname ILIKE '%ov';
+
+-- Find the name of the most expensive item for purchase,
+-- in the name of which there is the letter 'V', but not the first and not the last one.
+SELECT
+  products.name,
+  MAX(products.purchase_price)
```
Comment (ivshapovalov) [inline src/main/resources/sql/select2-1-select2-5.sql]: Group is redundant.

File: src/main/resources/sql/select2-1-select2-5.sql [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+WHERE products.purchase_price = (SELECT MAX(purchase_price)
+                          FROM products
+                          WHERE name ~* '^([^v])+([v]).+([^v])$')
+GROUP BY products.name;
+
+--Find the names of buyers whose names contain the letter 'V' and no more than two times
+--For a regular expression, thanks to @kortov
+SELECT c.name
+FROM customers c
+WHERE c.name SIMILAR TO '([^Vv]*[Vv][^Vv]*){0,2}';
```
Comment (ivshapovalov) [inline src/main/resources/sql/select2-1-select2-5.sql:26]: Good regex

**Review by ivshapovalov — CHANGES_REQUESTED** [review]

File: src/main/resources/sql/populate.sql (hunk 1 above)
Comment (AVM82) [reply src/main/resources/sql/populate.sql:12]: Thanks for your comment, but I have a boundary value for the test. It's 16 and 35. I think that's enough

File: src/main/resources/sql/select2-1-select2-5.sql (hunk 2 above)
Comment (AVM82) [reply src/main/resources/sql/select2-1-select2-5.sql:2]: The task is asked to find the names of customers

File: src/main/resources/sql/select2-1-select2-5.sql (hunk 3 above)
Comment (AVM82) [reply src/main/resources/sql/select2-1-select2-5.sql:7]: The task is asked to find the names of customers

File: src/main/resources/sql/select2-1-select2-5.sql (hunk 4 above)
Comment (AVM82) [reply src/main/resources/sql/select2-1-select2-5.sql]: Of course!
I fixed this.

File: src/main/resources/sql/select2-1-select2-5.sql (hunk 5 above)
Comment (AVM82) [reply src/main/resources/sql/select2-1-select2-5.sql:26]: Thanks to @kortov

**Review by ivshapovalov — APPROVED** [review]
